lib/src/test/java/vmath/Mat4Test.java

```diff
@@ -2,10 +2,33 @@
 
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
 public class Mat4Test {
 
     @Test
     void givenNoArgs_whenMatrix4Created_thenMatrixIsIdentity() {
-        //TODO(): check default constructor for identity matrix using get() method
+        Mat4 m = new Mat4();
+
+        for (int i = 0; i < 4; i++) {
+            for (int j = 0; j < 4; j++) {
+                if (i == j) {
+                    assertEquals(m.get(i,j), 1.0);
```

Comment:
Add one whitespace after the `,` operator. Make the `1.0` a `float` constant literal by adding `f` at the end.